benchto-driver/src/main/java/io/trino/benchto/driver/listeners/BenchmarkServiceExecutionListener.java

```diff
@@ -138,6 +138,13 @@ public Future<?> benchmarkFinished(BenchmarkExecutionResult benchmarkExecutionRe
                     if (benchmarkExecutionResult.getUtcEnd() != null) {
                         builder.withEndTime(benchmarkExecutionResult.getUtcEnd().toInstant());
                     }
+                    // Throughput tests have a different query in every execution, but only one, aggregated execution is saved
+                    // so don't save statements for them.
```

Comment:
What is aggregated execution for throughput test?

Reply:
Throughput benchmarks save one execution for all queries executed, with an attribute with a list of query names that were executed (in order) and the number of executed queries as measurements. I called this an aggregated execution since it represents multiple queries. Regular benchmarks save one execution per query.